### DISCUSSION TITLE
"inot.top"（绳网网页版）已停止访问

### DISCUSSION THREAD

**Discussion by Yizakl** [share121/inter-knot#5573] [公告]
# https://www.inot.top 
此网址已停止访问
APP端未受影响
（https://share121.github.io/inter-knot 
会跳转到
https://www.inot.top/）

**Comment by fzdqz** [comment]
停止访问了吗？<img src="https://upload-bbs.miyoushe.com/upload/2023/03/09/7cb87b286c3d062300d31224508c0c9e_8768116406867668880.png" alt="丹恒 惊" width="75" >那我是进的哪个网页回的你？

**Reply by Yizakl** [reply]
是Github喵

**Comment by IHushuan** [comment]
![1728484572125](https://github.com/user-attachments/assets/9e6f8ca6-9a4a-49b8-9281-f080d679e67e)
可以啊

**Reply by Yizakl** [reply]
PWA发力了
甚至不是新版

**Comment by Myitian** [comment]
你说得对
inot.top 已经没有 DNS 记录
现在能打开的都是因为之前打开过，浏览器缓存了 PWA 页面

**Comment by Eric-h-L** [comment]
哦，怪不得我最近想进绳网一直进不去

**Comment by Muci-Rotterdam** [comment]
PC端好像inot.top还是上不去【241019 18:58  存档】

**Reply by Yizakl** [reply]
好像去掉，现在就是上不去（

**Comment by yandujun363** [comment]
2024/11/10  
通过修改hosts+nginx反向代理成功访问，并解决跨域问题~~虽然卡成PPT~~

**Reply by Yizakl** [reply]
大佬

**Reply by KELERSAMA** [reply]
大佬
